### ISSUE TITLE
Customize Code using Handlebars option is not shown VS2017

### ISSUE THREAD

**Issue by ErikEJ** [ErikEJ/EFCorePowerTools#7]
_From @ydlkrishna on May 19, 2018 4:18_

I am using VS2017 15.6.5 and EF Core Power Tools was working fine until couple of days back. Now, when I try to reverse engineer, the option "Customize Code using Handlingbars...." is not shown.

Any help on this highly appreciated....

![image](https://user-images.githubusercontent.com/16814709/40264821-01ee59ae-5b4a-11e8-81ea-21aadda3d6f5.png)

### Steps to reproduce
Right click the Class library --> EF Core Power Tools --> Reverse Engineer --> Select SQL Server --> Select tables --> Customize code using handlebars option is missing (Please find attached screenshot)

### Further technical details
Toolbox/Power Tools version: 4.7.568.0

Database engine: SQL Server 2017

Visual Studio or SSMS version: Visual Studio 2017 15.6.5

_Copied from original issue: ErikEJ/SqlCeToolbox#713_

**Comment by ErikEJ** [comment]
Yes, it is temporarily remove while moving to EF Core 2.1 - please let me know if you really need it, and I will dig out an older build (remember to disable auto update)

**Comment by ydlkrishna** [comment]
Hi @ErikEJ , yes it's very much needed for my current project development. It would be a great help, if you can get it back.
And also, is there any way to overwrite the existing models when we modify the models by select/unselect the tables?
Thanks,
Krishna

**Comment by funky81** [comment]
Hi ErickJ, do you any guides how to install from the source ? Please share if you have. Thanks

**Comment by ErikEJ** [comment]
Just install the attached .zip file

**Comment by ErikEJ** [comment]
Closing as duplicate

**Comment by ErikEJ** [comment]
@ydlkrishna Handlebars support is back in latest daily build
